orchagent/neighorch.h

```diff
@@ -74,6 +74,8 @@ class NeighOrch : public Orch, public Subject, public Observer
     bool addInbandNeighbor(string alias, IpAddress ip_address);
     bool delInbandNeighbor(string alias, IpAddress ip_address);
 
+    bool resolveNeighborEntry(const NeighborEntry &, const MacAddress &);
```

Comment:
We should enhance this function to have nbrs cache (to track nbr resolution in progress) to avoid any nbr resolution request for the same nbrs again and again.

Reply:
also, I see that we update the APP_DB with the nbr to resolved but we're deleting it when nbr is successfully resolved, please take care of the deletion if not done already.

Reply:
Adding a logic to track the neighbor resolution progress and delete entries from APPL_DB once done.

Reply:
ok